magento/magento2
# All images height issue

### Description (*)


### Fixed Issues (if relevant)

1. magento/magento2#25043: Since 2.3.3 Store logo automatic resize on mobile/smaller displays does not respect aspect ratio

### Manual testing scenarios (*)

1. Magento 2.3.3 
2. Upload images from backend (You can upload logo or images from cms block)
3. You can able to see the stretching of all Frontend images. 

In Magento 2.3.3 latest version there is an image height issue, It was proper working in the previous version so I have done changes as per the previous version.

### Questions or comments




### Contribution checklist (*)
